### ISSUE TITLE
Please Support .NET 8 SDK 

### ISSUE THREAD

**Issue by zengrd** [Cysharp/UniTask#592]
Now The lastest unity editor support .net 8, Please author can support .net 8 SDK

**Comment by timcassell** [comment]
Where did you hear about that? They've been working on it for 2+ years, but nothing about public availability was mentioned in the mega thread on the forums.

**Comment by zengrd** [comment]
Thank you for your response. I am encountering an issue after exporting my Unity project to WebGL using the dotnet8 SDK. The error message I receive is related to the Cysharp.Threading.Tasks.UniTask type, specifically a failure in setting up the VTable for the Awaiter[T] type, which results in the get_IsCompleted method not being found.

After some investigation, I suspect that this issue might be due to UniTask not supporting dotnet8. If anyone has encountered a similar problem or has any alternative solutions to resolve this issue, your help would be greatly appreciated.

Thank you in advance for your assistance!


> dotnet.native.8.0.0.13nu67fark.js? [sm]:3689 TypeLoadException: Could not find method 'get_IsCompleted' due to a type load error: VTable setup of type Cysharp.Threading.Tasks.UniTask`1+Awaiter[T] failed assembly:UniTask.dll type:Awaiter member:(null)
>    at LoadDll.InitHotUpdate(String filename)
>    at LoadDll.Start()
> UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
> UnityEngine.DebugLogHandler:LogException(Exception, Object)
> UnityEngine.Logger:LogException(Exception, Object)
> UnityEngine.Debug:LogException(Exception)
> UnityEngine.<>c:<RegisterUECatcher>b__0_0(Object, UnhandledExceptionEventArgs)
> Cysharp.Threading.Tasks.CompilerServices.AsyncUniTaskMethodBuilder:Start(<InitHotUpdate>d__2&)
> LoadDll:InitHotUpdate(String)
> LoadDll:Start()
> 
> (env: macOS,mg,1.06.2405010; lib: 3.2.5)
> _JS_Log_Dump2 @ dotnet.native.8.0.0.13nu67fark.js? [sm]:3689
> WebGLPrintfConsolev(LogType, char const*, void*) @ 2aa7b7c6:0xa6aaee
> InternalErrorConsole(char const*, ...) @ 2aa7b7c6:0xa6c4a8
> DebugStringToFilePostprocessedStacktrace(DebugStringToFileData const&) @ 2aa7b7c6:0xa6c162
> DebugStringToFile(DebugStringToFileData const&) @ 2aa7b7c6:0xa6b7c6
> Scripting::LogExceptionFromManaged(ScriptingExceptionPtr, int, char const*, bool, Scripting::LogExceptionFromMangedSettings const*) @ 2aa7b7c6:0xa09a7d
> DebugLogHandler_CUSTOM_Internal_LogException(ScriptingBackendNativeObjectPtrOpaque*, ScriptingBackendNativeObjectPtrOpaque*) @ 2aa7b7c6:0xab5b60
> _do_icall @ 2aa7b7c6:0xb18209
> do_icall_with_exception @ 2aa7b7c6:0xb35bec
> do_icall @ 2aa7b7c6:0xb17d9c
> do_icall_wrapper @ 2aa7b7c6:0xb135d0
> mono_interp_exec_method @ 2aa7b7c6:0xb05d49
> interp_runtime_invoke @ 2aa7b7c6:0xb14c26
> mono_jit_runtime_invoke @ 2aa7b7c6:0xc0cb12
> do_runtime_invoke @ 2aa7b7c6:0xbd2e53
> mono_runtime_try_invoke @ 2aa7b7c6:0xbd3741
> mono_runtime_try_invoke_handle @ 2aa7b7c6:0xbd61f5
> mono_runtime_delegate_try_invoke_handle @ 2aa7b7c6:0xbd7d75
> mono_unhandled_exception_checked @ 2aa7b7c6:0xbd7a90
> mono_unhandled_exception_internal @ 2aa7b7c6:0xbd7839
> mono_handle_exception_internal @ 2aa7b7c6:0xc1e449
> mono_handle_exception @ 2aa7b7c6:0xc2014d
> interp_throw @ 2aa7b7c6:0xb1761b
> interp_throw_ex_general @ 2aa7b7c6:0xb130e2
> mono_interp_exec_method @ 2aa7b7c6:0xb12d71
> interp_runtime_invoke @ 2aa7b7c6:0xb14c26
> mono_jit_runtime_invoke @ 2aa7b7c6:0xc0cb12
> do_runtime_invoke @ 2aa7b7c6:0xbd2e53
> mono_runtime_try_invoke @ 2aa7b7c6:0xbd3741
> mono_runtime_invoke @ 2aa7b7c6:0xbd5ecf
> scripting_method_invoke(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingArguments&, ScriptingExceptionPtr*, bool) @ 2aa7b7c6:0xa1254c
> ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool) @ 2aa7b7c6:0xa0bf34
> MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingExceptionPtr*) @ 2aa7b7c6:0xfca4e
> MonoBehaviour::InvokeMethodOrCoroutineChecked(ScriptingMethodPtr, ScriptingObjectPtr) @ 2aa7b7c6:0xfbb9b
> MonoBehaviour::DelayedStartCall(Object*, void*) @ 2aa7b7c6:0xfde38
> DelayedCallManager::Update(int) @ 2aa7b7c6:0x81e52
> InitPlayerLoopCallbacks()::EarlyUpdateScriptRunDelayedStartupFrameRegistrator::Forward() @ 2aa7b7c6:0x9639ee
> ExecutePlayerLoop(NativePlayerLoopSystem*) @ 2aa7b7c6:0x959fba
> ExecutePlayerLoop(NativePlayerLoopSystem*) @ 2aa7b7c6:0x95a076
> PlayerLoop() @ 2aa7b7c6:0x95a2e5
> MainLoop() @ 2aa7b7c6:0x873426
> dynCall_v @ 2aa7b7c6:0xd6532a
> browserIterationFunc @ dotnet.native.8.0.0.13nu67fark.js? [sm]:14864
> _callUserCallback @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12567
> runIter @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12955
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12869
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> setTimeout (async)
> wxRequestAnimationFrame @ gamePage.html:60
> window.requestAnimationFrame @ fix.js? [sm]:71
> (anonymous) @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13279
> requestAnimationFrame @ dotnet.native.8.0.0.13nu67fark.js? [sm]:13277
> Browser_mainLoop_scheduler_rAF @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12754
> Browser_mainLoop_runner @ dotnet.native.8.0.0.13nu67fark.js? [sm]:12883
> (anonymous) @ fix.js? [sm]:72
> cbWithTimeStamp @ gamePage.html:58
> Show 18 more frames
> dotnet.native.8.0.0.13nu67fark.js? [sm]:3689 TypeLoadException: Could not find method 'get_IsCompleted' due to a type load error: VTable setup of type Cysharp.Threading.Tasks.UniTask`1+Awaiter[T] failed assembly:UniTask.dll type:Awaiter member:(null)
>    at LoadDll.InitHotUpdate(String filename)
>    at LoadDll.Start()
> 
> (env: macOS,mg,1.06.2405010; lib: 3.2.5)

**Comment by neuecc** [comment]
I do not know about .NET 8 Support for Unity.
First, can you explain "my Unity project to WebGL using the dotnet8 SDK"?
Please give me some screenshots and the URL of the documentation if you have it.

**Comment by zengrd** [comment]
> I do not know about .NET 8 Support for Unity. First, can you explain "my Unity project to WebGL using the dotnet8 SDK"? Please give me some screenshots and the URL of the documentation if you have it.

Hello,

I have read through the issue and I believe I have found the source of the problem. It seems that the issue was caused by the specific version of the Unity engine used in China, which defaults to using .NET 8. Fortunately, I have managed to resolve this issue on my end.

I sincerely apologize for any confusion or inconvenience caused by the error report, which was not due to UniTask itself. I appreciate your patience and understanding.

I will now proceed to close this thread. Thank you for your attention and support.
